README.md

```diff
@@ -37,57 +37,94 @@ Example:
+  summary: 'The world is closer than ever with over 100 languages',
+  installs: '500,000,000+',
+  minInstalls: 500000000,
+  score: 4.482483,
   scoreText: '4.5',
-  ratings: 386,
-  reviews: 168,
-  histogram: { '1': 14, '2': 11, '3': 26, '4': 51, '5': 284 },
+  ratings: 6811669,
+  reviews: 1614618,
+  histogram: { '1': 370042, '2': 145558, '3': 375720, '4': 856865, '5': 5063481 },
   price: 0,
   free: true,
   currency: 'USD',
   priceText: 'Free',
   offersIAP: false,
   IAPRange: undefined,
-  size: '34M',
-  androidVersion: '2.3',
-  androidVersionText: '2.3 and up',
-  developer: 'DxCo Games',
-  developerId: 'DxCo+Games',
-  developerEmail: 'dxcogames@gmail.com',
-  developerWebsite: 'http://www.dxco-games.com/',
-  developerAddress: undefined,
-  genre: 'Action',
-  genreId: 'GAME_ACTION',
+  size: 'Varies with device',
+  androidVersion: 'VARY',
+  androidVersionText: 'Varies with device',
+  developer: 'Google LLC',
+  developerId: '5700313618786177705',
+  developerEmail: 'translate-android-support@google.com',
+  developerWebsite: 'http://support.google.com/translate',
+  developerAddress: '1600 Amphitheatre Parkway, Mountain View 94043',
+  privacyPolicy: 'http://www.google.com/policies/privacy/',
+  developerInternalID: '5700313618786177705',
+  genre: 'Tools',
+  genreId: 'TOOLS',
   familyGenre: undefined,
   familyGenreId: undefined,
-  icon: 'https://lh6.ggpht.com/5mI27oolnooL__S3ns9qAf_6TsFNExMtUAwTKz6prWCxEmVkmZZZwe3lI-ZLbMawEJh3',
-  headerImage: 'https://lh4.ggpht.com/kKfRICvVTCikV4MLqsP0kWEth2F-I1Qt4jxMdklOdE2r8AmtrE-Umn6_WH_cGExXnjk-',
-  screenshots: [ 'https://lh5.ggpht.com/gD8L81t4CFKI21aOVkSnfVHioInwnt0XxMMWA-dBB2aU5bk3UfxGn8Hcq_KxcM6m430'],
-  video: 'https://www.youtube.com/embed/PFGj-W8Pe5s?ps=play&vq=large&rel=0&autohide=1&showinfo=0',
-  videoImage: 'https://i.ytimg.com/vi/PFGj-W8Pe5s/hqdefault.jpg',
-  contentRating: 'Mature 17+',
-  contentRatingDescription: 'Violence, Blood and Gore',
-  adSupported: true,
-  released: 'Feb 27, 2015',
-  updated: 1432677999000,
-  version: '1.4',
-  recentChanges: '- Added a hint system<br>- Added share option in level finished',
-  comments: [ 'Great!', 'PvZ', 'LOL', 'Zombie killer', 'Una pasada' ]
-  }
+  icon: 'https://lh3.googleusercontent.com/ZrNeuKthBirZN7rrXPN1JmUbaG8ICy3kZSHt-WgSnREsJzo2txzCzjIoChlevMIQEA',
+  headerImage: 'https://lh3.googleusercontent.com/e4Sfy0cOmqpike76V6N6n-tDVbtbmt6MxbnbkKBZ_7hPHZRfsCeZhMBZK8eFDoDa1Vf-',  screenshots: [
+    'https://lh3.googleusercontent.com/dar060xShkqnJjWC2j_EazWBpLo28X4IUWCYXZgS2iXes7W99LkpnrvIak6vz88xFQ',
+    'https://lh3.googleusercontent.com/VnzidUTSWK_yhpNK0uqTSfpVgow5CsZOnBdN3hIpTxODdlZg1VH1K4fEiCrdUQEZCV0',
+    'https://lh3.googleusercontent.com/zo76JEdXGsBcxbsJXX3XPN0pGicjIYAy7mi95al8LTT14H4xrer-lE2kR_DEXCn-GY_l',
+    'https://lh3.googleusercontent.com/8HwziIPNKGnrJe5l6MfdAm6XeZ7nYEWjLXli34D5xyzBT9jE1qIQstzZXJP9X63b4A',
```

Comment:
same with this, this object is expected to be illustratory, there's no benefit in showing 20 images vs 4, just makes the readme longer